x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/settings/product_documentation/index.tsx

```diff
@@ -72,11 +54,18 @@ export const ProductDocumentationManagement: React.FC = React.memo(() => {
         </EuiFlexItem>
       </EuiFlexGroup>
     );
-  }, [isInstalling, isStatusLoading, onClickInstall]);
+  }, [isInstalling, onClickInstall]);
 
-  if (isInstalled) {
+  // The last condition means that the installation was started by the plugin
+  if (
+    !status ||
```

Comment:
When is `status` actually `undefined`? Just on startup before any install action or check is performed?

Reply:
`!status` when `useKnowledgeBaseStatus` wasn't run yet